pkg/sloop/storemanager/storemanager.go

```diff
@@ -165,9 +166,8 @@ func (sm *StoreManager) Shutdown() {
 }
 
 func (sm *StoreManager) refreshStats() *storeStats {
-	// On startup we have 2 routines trying to do this at the same time
 	// If we have fresh results its good enough
-	if sm.stats != nil && time.Since(sm.stats.timestamp) < time.Second {
```

Comment:
Are we removing this condition since `sm.stats` will always be `!= nil `?

Reply:
Correct, even `generateStats()` returns a `storeStats{}` struct.
The default values of the `storeStats{}` struct can be safely used, such as `timestamp`.